app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsActivity.java

```diff
@@ -523,6 +530,7 @@ protected void addItemRow(ArrayObjectAdapter parent, ItemRowAdapter row, int ind
 
     protected void addAdditionalRows(ArrayObjectAdapter adapter) {
         Timber.d("Item type: %s", mBaseItem.getBaseItemType().toString());
+        boolean chaptersRowEnabled;
```

Comment:
You can just assign it here
```suggestion
        boolean chaptersRowEnabled = userPreferences.getValue().get(UserPreferences.Companion.getChaptersRowEnabled());
```